mongo/integration/change_stream_test.go

```diff
@@ -586,6 +586,23 @@ func TestChangeStream_ReplicaSet(t *testing.T) {
 			assert.Equal(mt, 2, numClearedEvents, "expected two PoolCleared events, got %d", numClearedEvents)
 		})
 	})
+	mt.RunOpts("call to cursor.Next after cursor closed", mtest.NewOptions().MinServerVersion("4.0"), func(mt *mtest.T) {
```

Comment:
Some more whitespace would make this more readable. Generally, try to separate things into logical blocks. For example this test could look like:

```
setup change stream and defer the close

generate events (both the generateEvents call and the Coll.Drop)

iterate stream in loop

assert last event's operationType is "invalidate" and Next returns false
```

Reply:
Also, add a comment to explain why the min server version is 4.0.

Reply:
done